setup-production.adoc

```diff
@@ -340,33 +295,35 @@ JIRA_IMPORTER_CERT: "cert-from-jira"
 JIRA_IMPORTER_PUB_CERT: "pub-cert-from-jira"
 ----
 
-**Service: taiga-front**
+Service: `taiga-front`
 [source, bash]
 ----
 ENABLE_JIRA_IMPORTER: "true"
 ----
 
-.Trello importer
+.**Enable the Trello importer**
 
-**Service: taiga-back**
+Service: `taiga-back`
 [source, bash]
 ----
 ENABLE_TRELLO_IMPORTER: "True"
 TRELLO_IMPORTER_API_KEY: "api-key-from-trello"
 TRELLO_IMPORTER_SECRET_KEY: "secret-key-from-trello"
 ----
 
-**Service: taiga-front**
+Service: `taiga-front`
 [source, bash]
 ----
 ENABLE_TRELLO_IMPORTER: "true"
 ----
 
-=== Advanced configuration and customisation
+=== Advanced configuration
+
+The advanced configuration **will ignore** the environment variables in `docker-compose.yml` or `docker-compose-inits.yml`.
```

Comment:
I'd add: "Skip this section if you're using env vars."

Reply:
Nice, I'll add it.